cocotb_build_libs.py

```diff
@@ -501,7 +501,7 @@ def _get_common_lib_ext(include_dirs, share_lib_dir):
     if os.name == "nt":
         libcocotbutils_sources += ["libcocotbutils.rc"]
     libcocotbutils_libraries = ["gpilog"]
-    if os.name != "nt":
+    if os.name != "nt" and not "bsd" in sys.platform:
```

Comment:
I think positive conditions read better. Feel free to ignore if this is problematic.
```suggestion
    if sys.platform.startswith(("linux", "darwin", "cygwin", "msys"))
```